pom-jna-platform.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>net.java.dev.jna</groupId>
   <artifactId>jna-platform</artifactId>
-  <version>4.1.0</version>
+  <version>4.1.1-SNAPSHOT</version>
```

Comment:
These should not be changed part of this PR, they are not material to this change.